core/drivers/crypto/crypto_api/include/drvcrypt_hash.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * Brief   Hash interface calling the HW crypto driver.
+ */
+#ifndef __DRVCRYPT_HASH_H__
+#define __DRVCRYPT_HASH_H__
+
+#include <tee_api_types.h>
+
+/*
+ * Crypto Library Hash driver operations
+ */
+struct drvcrypt_hash {
+	/* Allocates of the Software context */
```

Comment:
s/of the/the/

Some suggested rephrasing:
"Allocate operation context"
"Free operation context"
"Initialize operation context"
"Update  operation context"
"Finalize  operation context"
"Copy operation context"
...or maybe even no inline comment.